source/Ship.h

```diff
@@ -380,6 +380,12 @@ class Ship : public Body, public std::enable_shared_from_this<Ship> {
 	double MaxVelocity() const;
 	double ReverseAcceleration() const;
 	double MaxReverseVelocity() const;
+	// This is their potential acceleration right now
+	double TrueAcceleration() const;
+	// This is their potential turn right now
+	double TrueTurnRate() const;
+	// This is their current speed right now
+	double CurrentSpeed() const;
```

Comment:
```suggestion
	// Calculate the ship's current maximum acceleration and turn rate, accounting for slowness.
	double TrueAcceleration() const;
	double TrueTurnRate() const;

	// The ship's current speed right now
	double CurrentVelocity() const;
```

Reply:
Comments were condensed into one and revised. Please take a second look.

Reply:
The comments look good.